mod-audit/src/test/java/org/folio/ModAuditApiTest.java

```diff
@@ -29,4 +29,9 @@ public void tearDown() {
   public void testGetData() {
     runFeatureTest("audit-data");
```

Comment:
Please remove audit-data feature, as this one is never used.

Reply:
Removed, thank you.